src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,27 @@
+import * as dateFns from 'date-fns'
+import { UNIX_TIME_START, TIME_SEPARATOR } from '../common'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const checkHourExist = (hours: string): boolean => {
```

Comment:
I think we need to move it to a separate file

Reply:
I think that's a moot point. Because it not used nowhere and our project not big, and better to avoid a large filesystem structure.